ironfish/src/network/messageRouters/globalRpc.test.ts

```diff
@@ -162,7 +150,11 @@ describe('Global Rpc', () => {
     await expect(promise).toRejectErrorInstance(CannotSatisfyRequestError)
   })
 
-  it('throws when peers available but none respond', async () => {
+  // TODO: It looks like in Jest 27.0.3, there is a regression introduced when
+  // using `setTimeout` and `setImmediate`. We should re-enable this test once
+  // that regression has been addressed.
+  // eslint-disable-next-line jest/no-disabled-tests
```

Comment:
Is there a Jest issue for this? Only asking because it's possible we're doing something hacky here that they actually fixed. I remember having to mess with this quite a bit to get it to pass as expected

Reply:
I think I'm running into a similar issue described [here](https://github.com/facebook/jest/issues/11607). I'm happy to re-factor this test offline if you think there's a better way we can handle this case.

Reply:
Ah, nice find. If you want to file a separate issue to investigate refactoring this test, we can address that in a separate branch.

Reply:
Hey, I looked deeply at this problem and it seems like this version of jest is maybe buggy and unstable. There's also no link to the issue here. If setTimeout and setInterval are buggy and broken, it seems like maybe we shouldn't upgrade to them. If something as core as their microtask queue is broken, who knows what else is going wrong. Should we wait to update til this can get fixed, or another version comes out then we can try that?

Reply:
We reverted jest here as the resolution.